demo-vue/README.md

```diff
@@ -0,0 +1,23 @@
+# demo-vue
+
+> A Vue.js project demostrating the Strickland validation framework
+
+## Build Setup
+
+``` bash
+# yarn or npm
+
+# install dependencies
+yarn install
+
+# serve with hot reload at localhost:8080
+yarn run dev
+
+# build for production with minification
+yarn run build
+
+# build for production and view the bundle analyzer report
+yarn run build --report
+```
```

Comment:
How would you feel about aligning this to `npm` instead of `yarn`?  We can also add a root-level script for `npm run demo-vue`.

Reply:
Yea sure thing. I've just started transitioning to yarn but we can make this npm first and if people want to use yarn we leave the yarn.lock files in for them too.

Reply:
At some point it may make sense to move to a "demos" folder holding "demo-vue" and "demo-react" but that can wait until everything is stable for a quick shuffle. Would help people easily identify the react demo, rather rely on them assuming it's the default.